api/src/main/java/org/openmrs/module/htmlformentryui/tag/UiIncludeTagHandler.java

```diff
@@ -64,9 +70,20 @@ public boolean doStartTag(FormEntrySession formEntrySession, PrintWriter out, No
         }
 
         String fragment = getAttribute(node, "fragment", null);
+		
         if (StringUtils.isNotEmpty(fragment)) {
+        	String params = null;
+			// Check whether we have fragment parameters
+        	if (StringUtils.contains(fragment, '?')) {
```

Comment:
Use Apache's URL/URI utils to extract params from an URI-looking string. You have to look around for those.